sru-templates/manual/azure-sru

```diff
@@ -15,11 +17,42 @@ RESOURCE_GROUP="<specify your resource group>"
 BOOT_DIAG="<specify your boot diagnostics group>"
 SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
+
+IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+
+if [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; then
+	echo "Account is not yet registered for AllowIPv6VirtualNetwork or AllowIPv6CAOnStandardLB. Can take 30 mins. Activate feature now (y/n)?"
+    read RESP
+    if [ "$RESP" = "y" ]; then
+        az feature register --name AllowIPv6VirtualNetwork --namespace Microsoft.Network
+        az feature register --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network
+    fi
+    echo "Waiting for AllowIPv6VirtualNetwork freature registration"
+    while [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; do
+        echo -n "."
+        sleep 5
+        IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+        IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+    done
+fi
+
+IPV6_REGISTRATION_COMPLETE=`az provider show -n Microsoft.Network | jq -r ".registrationState"`
+echo "Waiting for Microsoft.Network IPv6 feature registration"
+while [ "$IPV6_REGISTRATION_COMPLETE"  != "Registered" ]; do
+    echo -n "."
+    sleep 5;
```

Comment:
```suggestion
    sleep 5
```

Reply:
This doesn't look resolved?

Reply:
again lost because I accepted commit suggestion in UI, but then didn't pull down the changes from 'upstream'. sorry about that. doing this locally and will push up.